.github/workflows/certification.yml

```diff
@@ -93,6 +93,8 @@ jobs:
           required-secrets: AzureServiceBusConnectionString
         - component: bindings.azure.cosmosdb
           required-secrets: AzureCosmosDBUrl,AzureCosmosDB,AzureCosmosDBCollection,AzureCosmosDBMasterKey,AzureCertificationTenantId,AzureCertificationServicePrincipalClientId,AzureCertificationServicePrincipalClientSecret
+        - component: bindings.azure.eventhubs
+          required-secrets: AzureBlobStorageAccount,AzureBlobStorageAccessKey,AzureEventHubsBindingsHub,AzureEventHubsBindingsNamespace,AzureCertificationServicePrincipalClientId,AzureCertificationTenantId,AzureCertificationServicePrincipalClientSecret,AzureResourceGroupName,AzureCertificationSubscriptionId,AzureEventHubsBindingsContainer
```

Comment:
You will also need `AzureEventHubsBindingsConnectionString` because in addition to service principal authentication you also need to test the connection method with a connection string.